packages/sku/lib/openBrowser/index.js

```diff
@@ -2,32 +2,61 @@
 // https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/openBrowser.js
 
 const execSync = require('node:child_process').execSync;
-const defaultBrowser = require('x-default-browser');
 const open = require('open');
 
 const isCI = require('../isCI');
 
-module.exports = (url) => {
+const OSX_CHROME = 'google chrome';
+
+module.exports = async (url) => {
   if (process.env.OPEN_TAB !== 'false' && !isCI) {
-    defaultBrowser((err, res) => {
-      const useChrome = err ? false : res.isChrome;
```

Comment:
I'm experiencing incorrect behaviour when I have both Firefox (my default) and Chrome open (no existing tab). I'd expect Firefox to get the new tab, but instead Chrome gets it.

I think the logic needs to be something like this:
1. Check if default browser is one of the supported chromium browsers.
2. If it is, do the new logic you've implemented
3. If it's not, check if the default browser is currently open with:
```
execSync(`ps cax | grep "${defaultBrowser}"`);
```
4. If that fails, it'll hit a `catch`, at which point you'll want to try doing the Chromium stuff.
5. If _that_ fails, it should fall through to `open`

This might require some abstracting of chromium browser check and some loop shenanigans.

Reply:
Ahh yes, I have not been testing with non-Chromium. Will fix this